client/my-sites/post-type-list/post-actions-ellipsis-menu/share.jsx

```diff
@@ -44,11 +44,12 @@ class PostActionsEllipsisMenuShare extends Component {
 	}
 
 	render() {
-		const { translate, status, isPublicizeEnabled: isPublicizeEnabledForSite } = this.props;
+		const { translate, status, type, isPublicizeEnabled: isPublicizeEnabledForSite } = this.props;
 		if (
 			! config.isEnabled( 'posts/post-type-list' ) ||
 			! includes( [ 'publish' ], status ) ||
-			! isPublicizeEnabledForSite
+			! isPublicizeEnabledForSite ||
+			type.name !== 'post'
```

Comment:
We've seen recently that this object can be `null` while loading. Let's just use the `post.type` string because we don't actually need any of the properties of the post type.

Should we go ahead and support `page` here? If not, we need to be sure to add that later.

Reply:
Good call re: using `post.type`. I've just pushed a change to switch to that.

Re: enabling for page. As we've discussed at grooming, since the current implementation of Site Pages doesn't contain "Share", and the justification for disabling this for CPTs is that we do not know the experience is optimized for sharing of CPTs, the same logic extends to pages. We should explicitly enable it when we move "Site Pages" to `PostTypeList`.